<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1471</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登陆</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/myimages/icon/登陆.png</normaloff>:/myimages/icon/登陆.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>40</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>26</pointsize>
     <weight>3</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 25 26pt &quot;3ds Light&quot;;</string>
   </property>
   <property name="text">
    <string>LOGIN</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="hotelButton">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>280</y>
     <width>261</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;</string>
   </property>
   <property name="text">
    <string>I'm a Hotel Manager</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="customerButton">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>280</y>
     <width>201</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;</string>
   </property>
   <property name="text">
    <string>I'm a Customer</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordInput">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>210</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);
hover 
{
border: 1px solid blue;
}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userNameInput">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>150</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>User name</string>
   </property>
  </widget>
  <widget class="QLabel" name="picLabel">
   <property name="geometry">
    <rect>
     <x>-80</x>
     <y>0</y>
     <width>981</width>
     <height>631</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icon.qrc">:/myimages/icon/hotel7.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>460</y>
     <width>341</width>
     <height>36</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;
border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>410</y>
     <width>341</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>12</pointsize>
     <weight>3</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;
border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>510</y>
     <width>341</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;
border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="SystemButton">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>340</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;</string>
   </property>
   <property name="text">
    <string>I'm the System Maneger</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>110</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 10pt &quot;3ds Light&quot;;</string>
   </property>
   <property name="text">
    <string>Login with your account</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>userNameInput</tabstop>
  <tabstop>passwordInput</tabstop>
  <tabstop>customerButton</tabstop>
  <tabstop>hotelButton</tabstop>
  <tabstop>SystemButton</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>registerButton</tabstop>
  <tabstop>exitButton</tabstop>
 </tabstops>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
